PocketKit/Sources/Analytics/Tracker/Tracker.swift

```diff
@@ -22,4 +22,8 @@ public extension Tracker {
     func childTracker(hosting context: UIContext) -> Tracker {
         childTracker(with: [context])
     }
+
+    func track(event: Event, filename: String = #file, line: Int = #line, column: Int = #column, funcName: String = #function) {
```

Comment:
This is how we add default values to protocols https://medium.com/@georgetsifrikas/swift-protocols-with-default-values-b7278d3eef22